flask_of_oil/opaque_validator.py

```diff
@@ -140,7 +138,4 @@ def validate(self, token):
         if 'scope' not in introspect_response:
             raise OpaqueValidatorException("Missing scope field in introspection response")
 
-        return {"subject": introspect_response['sub'],
-                "scope": introspect_response['scope'],
-                "cnf": introspect_response.get("cnf", ""),
-                "active": True}
+        return introspect_response
```

Comment:
strange that we didn't do this from the get-go. I wonder if there was some rationale for creating this object and using it above in the cached response.